draft-ietf-quic-multipath.md

```diff
@@ -1069,9 +1069,9 @@ MUST silently ignore the frame.
 ## ACK_MP Frame {#ack-mp-frame}
 
 The ACK_MP frame (types TBD-00 and TBD-01)
-is an extension of the ACK frame defined by {{QUIC-TRANSPORT}}. It is
+is an extension of the ACK frame specified in {{Section 19.3 of QUIC-TRANSPORT}}. It is
 used to acknowledge packets that were sent on different paths using
-multiple packet number spaces. If the frame type is TBD-01, ACK_MP frames
+each an own packet number space. If the frame type is TBD-01, ACK_MP frames
```

Comment:
I cannot create a suggestion because of the deleted line, but I would suggest instead

"used to acknowledge packets that were sent on different paths, as each
path has its own packet number space. If the frame type [...]"